test/form-builder/components/FormDetailContainer.spec.js

```diff
@@ -661,6 +662,46 @@ describe('FormDetailContainer', () => {
       }, 500);
     });
 
+    it('should not call save form name translation endpoint if the form is of 1st version' +
```

Comment:
@vinay-ThoughtWorks @bsneha90 can you brief the logic behind not to make save form name translation api call if the form is of 1st version.

Reply:
@vinay-ThoughtWorks thanks for the clarification